### ISSUE TITLE
Errors applying compatability patches for Acorns mod

### ISSUE THREAD

**Issue by PhyDoesStuff** [l33tmaan/vs-expandedfoods#16]
Mod version: v1.5.3-pre.11
Game version: 1.16.5

Looks like there is an issue with paths provided for the acornhardtack.json patch:

```
10.6.2022 18:44:21 [Error] Patch 4 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /2/enabled is invalid: The json path /2/enabled was not found. No such element '2' at the root path
10.6.2022 18:44:21 [Error] Patch 5 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /3/enabled is invalid: The json path /3/enabled was not found. No such element '3' at the root path
10.6.2022 18:44:21 [Error] Patch 6 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /4/enabled is invalid: The json path /4/enabled was not found. No such element '4' at the root path
10.6.2022 18:44:21 [Error] Patch 7 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /5/enabled is invalid: The json path /5/enabled was not found. No such element '5' at the root path
10.6.2022 18:44:21 [Error] Patch 8 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /6/enabled is invalid: The json path /6/enabled was not found. No such element '6' at the root path
10.6.2022 18:44:21 [Error] Patch 9 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /6/enabled is invalid: The json path /6/enabled was not found. No such element '6' at the root path
10.6.2022 18:44:21 [Error] Patch 10 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /7/enabled is invalid: The json path /7/enabled was not found. No such element '7' at the root path
10.6.2022 18:44:21 [Error] Patch 11 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /8/enabled is invalid: The json path /8/enabled was not found. No such element '8' at the root path
10.6.2022 18:44:21 [Error] Patch 12 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /9/enabled is invalid: The json path /9/enabled was not found. No such element '9' at the root path
10.6.2022 18:44:21 [Error] Patch 13 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /10/enabled is invalid: The json path /10/enabled was not found. No such element '10' at the root path
10.6.2022 18:44:21 [Error] Patch 14 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /11/enabled is invalid: The json path /11/enabled was not found. No such element '11' at the root path
10.6.2022 18:44:21 [Error] Patch 15 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /12/enabled is invalid: The json path /12/enabled was not found. No such element '12' at the root path
10.6.2022 18:44:21 [Error] Patch 16 (target: expandedfoods:recipes/kneading/acorns/acornhardtack.json) in expandedfoods:patches/compatibility/acorns/acornhardtack.json failed because supplied path /13/enabled is invalid: The json path /13/enabled was not found. No such element '13' at the root path```

**Comment by l33tmaan** [comment]
Should be fixed in 1.5.3.

**Comment by PhyDoesStuff** [comment]
Confirmed, no longer seeing in 1.5.3.
